reservation_units/utils/reservation_unit_pricing_helper.py

```diff
@@ -115,14 +116,12 @@ def calculate_vat_prices(cls, data: Dict[str, Any]) -> Dict[str, Any]:
                 pricing["highest_price"] = highest_price_net
                 pricing["lowest_price"] = lowest_price_net
             else:
-                pricing["highest_price"] = round(
-                    Decimal(highest_price_net * (1 + tax_percentage.decimal)),
-                    2,
-                )
-                pricing["lowest_price"] = round(
-                    Decimal(lowest_price_net * (1 + tax_percentage.decimal)),
-                    2,
-                )
+                pricing["highest_price"] = Decimal(
+                    highest_price_net * (1 + tax_percentage.decimal)
+                ).quantize(Decimal("1.0"), rounding=decimal.ROUND_HALF_UP)
```

Comment:
Original code had two decimals so I think this should be:
```suggestion
                ).quantize(Decimal("1.00"), rounding=decimal.ROUND_HALF_UP)
```

Same with the next `quantize`

Reply:
![](https://media.giphy.com/media/w89ak63KNl0nJl80ig/giphy.gif)